### PR TITLE
Add hook for non-const motion validator (for setting clearance)

### PR DESCRIPTION
**Pull request by davetcoleman**

### CODE REVIEW

**Comment by davetcoleman** [comment]
I don't see how this can hurt...

File: src/ompl/base/SpaceInformation.h [hunk 1]
```diff
@@ -180,6 +180,11 @@ namespace ompl
                 return motionValidator_;
             }
 
+            MotionValidatorPtr& getMotionValidatorNonConst()
```
Comment (mamoll) [inline src/ompl/base/SpaceInformation.h]: Just change the method name to `getMotionValidator`. The right version of the method will be called automatically.

Comment (davetcoleman) [inline src/ompl/base/SpaceInformation.h]: I learned the ``NonConst`` standard from [moveit](https://github.com/ros-planning/moveit/blob/d001746257005e7bde01f1bed581ab78a763d3f0/moveit_core/planning_scene/include/moveit/planning_scene/planning_scene.h#L156) but am happy to change it

Comment (mamoll) [inline src/ompl/base/SpaceInformation.h]: Please change. This is the convention in OMPL.

Comment (davetcoleman) [inline src/ompl/base/SpaceInformation.h]: done

**Pull request merged**